crypto/provider.c

```diff
@@ -22,7 +22,7 @@ OSSL_PROVIDER *OSSL_PROVIDER_try_load(OSSL_LIB_CTX *libctx, const char *name,
     int isnew = 0;
 
     /* Find it or create it */
-    if ((prov = ossl_provider_find(libctx, name, 0)) == NULL) {
+    if ((prov = ossl_provider_find(libctx, name, 1)) == NULL) {
```

Comment:
I don't understand this change. This seems to set `noconfig` to true, i.e. don't load config - which seems contrary to the title of this PR

Reply:
Yeah, this is not needed and does not make a sense. Dropped.